recipe/meta.yaml

```diff
@@ -10,15 +10,15 @@ source:
   sha256: 3e2f5826578e683999b93716285b3b1f344f157bf75fa9ce0a797564e742f062
 
 build:
-  noarch: python
```

Comment:
Skip py<36, see https://github.com/ipython/comm/blob/f4190da20bacd94d669819c725a7d0881dfe9c65/pyproject.toml#L11

Reply:
Thanks made an amendment

Reply:
@Arishamays1 Could you make one more minor change the `<` is missing.